### PR TITLE
[Enhancement] Get Device Info

### PR DESCRIPTION
**Pull request by ksmontville**
Added a new getDeviceInfo() function to utils.js

### CODE REVIEW

**Comment by ksmontville** [comment]
Task linked: [CU-868669jh6 Add device info to run level data](https://app.clickup.com/t/868669jh6)

File: src/utils.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import path from 'path-browserify';
 import mime from 'mime-types';
-import { deviceType, primaryInput } from 'detect-it';
+import {deviceType, primaryInput} from 'detect-it';
```
Comment (richford) [inline src/utils.js:3]: Do we need to `npm install detect-it`? Also, it doesn't appear that we use `deviceType` and `primaryInput` below. Did you want to add it to the output of `getDeviceInfo()`?

**Review by richford — CHANGES_REQUESTED** [review]

File: src/utils.js (hunk 1 above)
Comment (ksmontville) [reply src/utils.js:3]: Interesting, I didn't actually import those functions to use in my getDeviceInfo() commit. Perhaps they were left over from another utility function?

Comment (richford) [reply src/utils.js:3]: Oh I see, they were there before and your commit only changed the formatting. Thanks for clarifying. Do you think we should use any of the detect-it info in `getDeviceInfo` or do you think what you have already written covers all the bases?

**Review by richford — APPROVED** [review]

**Pull request merged**